libs/wingc/src/jsify.rs

```diff
@@ -1458,7 +1458,7 @@ impl<'a> JSifier<'a> {
 
 			// TODO Hack to make sure closures match the IInflight contract from wingsdk
 			if class_type.is_closure() {
-				code.line("_hash = require('crypto').createHash('md5').update(this._toInflight()).digest('hex');")
+				code.line(format!("_hash = {STDLIB_CORE}.closureId();"))
```

Comment:
`_hash` is currently being used to deduplicate handlers in various resources (like Api, Topic, etc.). So 2 inflights are "equal" for this purpose if their hashes are equal even if they are technically different instances.
The SDK tests are still passing because they're still generating hashes.

If we're going to change to this implementation (which I'm cool with), we should rename this field to `_id` or something and make sure we are using it consistently in the SDK tests. We also need to note that it's a global sequential id and shouldn't be used for anything expected to be stable between compilations. This is both because reordering the source changes it and because this id is globally shared in a possibly non-deterministic way.

Reply:
Got it, yeah I see the discrepancy now. Updating the name and the tests

Reply:
btw why was this changed in this PR? The other changes don't seem to conflict with the hash generation

Reply:
I've added a longer comment in the code about this 😄